### DISCUSSION TITLE
can't 'git clone' (only download with web browsers)

### DISCUSSION THREAD

**Discussion by dchmelik** [community/community#77523] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

For a month or more, I haven't been able to access GitHub with 'git clone'.  Traceroute shows no route to github.com: route ends somewhere on Microsoft's network other than github.com's IP address.  My Internet Service Provider (ISP) tested this and it's the current state on their own network and they also tested Comcast (perhaps USA's largest ISP) to find it's the case there.  I mentioned this on the oldest GNU/Linux's Internet Relay Chat (IRC) quasi-official channel (libera IRC ##slackware , which extra packages for Slackware depend on GitHub) where administrators/operators and other chatters found the same.  Strangely I can still access GitHub with Mozilla/Firefox & Konqueror/Chromium/Chrome but not always with Curl & Wget web browser that's mainly used to download a file such as source code.
        Am I blocked from 'git clone'?  Below is some example output that does basically the same; 'git clone' doesn't work but 'wget' works.

```
user@darkstar:~$ git clone https://github.com/ponce/slackbuilds
Cloning into 'slackbuilds'...
fatal: unable to access 'https://github.com/ponce/slackbuilds/': Could not resolve host: github.com
save/download error
user@darkstar:~$ wget https://github.com/Ponce/slackbuilds/archive/refs/tags/current-20231209.1.tar.gz
--2023-12-10 22:22:14--  https://github.com/Ponce/slackbuilds/archive/refs/tags/current-20231209.1.tar.gz
Resolving github.com (github.com)... 20.29.134.23
Connecting to github.com (github.com)|20.29.134.23|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://codeload.github.com/Ponce/slackbuilds/tar.gz/refs/tags/current-20231209.1 [following]
--2023-12-10 22:22:18--  https://codeload.github.com/Ponce/slackbuilds/tar.gz/refs/tags/current-20231209.1
Resolving codeload.github.com (codeload.github.com)... 20.29.134.24
Connecting to codeload.github.com (codeload.github.com)|20.29.134.24|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [application/x-gzip]
Saving to: ‘current-20231209.1.tar.gz’

current-20231209.1.tar.gz                [                           <=>                                       ]  19.95M  1.06MB/s    in 24s     

2023-12-10 22:22:47 (862 KB/s) - ‘current-20231209.1.tar.gz’ saved [20917910]

```

Update: now curl & wget usually don't work nor does it seem possible to download a .tar.gz of latest commits instead of .zip anymore.

**Comment by exilonium** [comment]
yeah i'm going through the same thing Using vpn helps
